components/search-tab-group/search-tab-group.jsx

```diff
@@ -9,26 +9,31 @@ import ProfileLoader from "../profile-loader/profile-loader.jsx";
 const TAB_NAMES = [`people`, `projects`];
 const DEFAULT_TAB_NAME = `projects`;
 
-class SearchTabGroup extends React.Component {
+// Extends React.PureComponent instead of React.Component as we want
+// this component to re-render only when there's change in its props or state.
+// Note that React.PureComponent runs *shallow comparison* to determine
+// if the component should be re-rendered or not. So make sure
+// SearchTabGroup's props and state doesn't contain anything mutatable otherwise
+// we will run into issues.
+class SearchTabGroup extends React.PureComponent {
   constructor(props) {
     super(props);
 
+    this.availableTabs = TAB_NAMES;
```

Comment:
This shouldn't have been a `state` in the first place 😬 Fixing this now.

Reply:
Why shouldn't it have been a state?

Reply:
We use `state` for variable whose values changes over the course of the component's lifetime. You can see in the file that `availableTabs" never gets changed so hence it shouldn't have been stored in `state`.

Reply:
We can rename this variable since "available" tabs can sound confusing and misleading.